style.css

```diff
@@ -150,9 +150,26 @@ main img {
         width: 50%;
         margin: 0 auto;
     }
+
 }
 
 .card a:visited,
 .card a:link {
     color: var(--blue);
-}
\ No newline at end of file
+}
+
+@media (hover:hover) {
+    .card:hover {
+        transform: rotate(0);
+    }
+
+    .card {
```

Comment:
Why are you duplicating all those rules? You already have most of them defined above, lines `83-89`. This should only contain what's different!

Reply:
You marked this as resolved but the duplicate rules are still here. I don't think you understand how a media query works. You don't need to duplicate the rules in order for them to apply.

Since you have most of them here, you should only add what's different.
<img width="542" alt="image" src="https://user-images.githubusercontent.com/11327761/200054571-4bb74523-e97f-4a27-8fc6-4b4ae2a4bdb9.png">

Reply:
Thank you for the feedback. Now I understand how media query works. 
I made the changes.